### ISSUE TITLE
Naming the homebridge devices independently

### ISSUE THREAD

**Issue by freezir12** [noelhibbard/homebridge-snapcast#2]
Could you add an option to name the snapcast devices independently from the ClientID or the name set in snapweb?

So that i could do this:
Name displayed in homebridge: Audio Livingroom
SnapclientID: OSMC_LivingRoom
Name set in Snapweb: OSMC_LivingRoom

Would be awesome! Thanks!

**Comment by noelhibbard** [comment]
This is possible but you can already rename any device in the Home app on your phone.

**Comment by freezir12** [comment]
Yep that is true. But would save some time in case of a reset of the phone, or losing the home data, or changing the home app, or doing a reinstall of the homebridge on my pi.
I just would like the option. Maybe you will find the time to implement this one day. :)
